scripts/install_all.sh

```diff
@@ -152,6 +152,18 @@ if ! command -v nomad > /dev/null; then
     fi
 fi
 
+if ! command -v black > /dev/null; then
+    echo "Installing black..."
+    if [ $BREW ]; then
+        $INSTALL_CMD black > $OUTPUT
```

Comment:
should we always use `pip3` for `black`?

Reply:
Ok thanks for testing that!! I've made that change.